browser/react/components/Login.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { login, logout } from '../../redux/reducers/auth';
+
+/* ----------------- COMPONENT ------------------ */
+
+class Login extends React.Component {
+  constructor (props) {
```

Comment:
I am too, but I'm not sure that's possible with the necessary mounting/unmounting that needs to be done.